content/automate/ManageIQ/System/Notification/Email.class/__class__.yaml

```diff
@@ -11,12 +11,52 @@ object:
     visibility: 
     owner: 
   schema:
+  - field:
+      aetype: assertion
+      name: sendmail
+      display_name: 
+      datatype: boolean
+      priority: 1
+      owner: 
+      default_value: 'true'
+      substitute: true
+      message: create
+      visibility: 
+      collect: 
+      scope: 
+      description: 
```

Comment:
@billfitzgerald0120 
Should we put a description that reads something like
"By default the email is enabled, to disable a specific email set this attribute to false"